slices/procps.yaml

```diff
@@ -0,0 +1,49 @@
+package: procps
+
+essential:
+  - procps_copyright
+
+slices:
+  bins:
+    essential:
+      - init-system-helpers_bins
+      - libc6_libs
+      - libncursesw6_libs
+      - libproc2-0_libs
+      - libsystemd0_libs
+      - libtinfo6_libs
+    contents:
+      /usr/bin/free:
+      /usr/bin/kill:
+      /usr/bin/pgrep:
+      /usr/bin/pidwait:
+      /usr/bin/pkill: {symlink: pgrep}
+      /usr/bin/pmap:
+      /usr/bin/ps:
+      /usr/bin/pwdx:
+      /usr/bin/skill:
+      /usr/bin/slabtop:
+      /usr/bin/snice: {symlink: skill}
```

Comment:
Same as before, where is this coming from? Does `postinst` installs these?

Reply:
```
dpkg -c procps_2%3a4.0.4-4ubuntu3_amd64.deb  | grep bin/snice
lrwxrwxrwx root/root         0 2024-03-31 08:16 ./usr/bin/snice -> skill
```

Reply:
Oka, thanks!
```suggestion
      /usr/bin/snice:
```